monitor-data/monitors.mdx

```diff
@@ -58,6 +59,33 @@ To create a threshold monitor, follow these steps:
 
 You have created a threshold monitor, and Axiom alerts you when the results from your query cross the threshold.
 
+## Anomaly monitors
+
+Anomaly monitors allow you to aggregate your event data and compare the results of this aggregation to what can be considered normal for the query. When the results are too much above or below the value that Axiom expects based on the event history, the monitor enters the alert state. The monitor remains in the alert state until the results no longer deviate from the expected value. An anomaly monitor sends you a notification each time it enters or exits the alert state.
+
+To create an anomaly monitor, follow these steps:
+
+1. Click the **Monitors** tab on the top of the page, and then click **New monitor**.
+2. Click **Anomaly monitor**.
+3. Name your monitor and add a description.
+4. Configure the monitor using the following options:
+    - The comparison operator is the rule to apply when comparing the results to the expected value. The possible values are **above**, **below**, and **above or below**.
+    - The tolerance factor controls the sensitivity of the monitor. It defines how much deviation from the expected value to accept without triggering the monitor. It’s a range above and below the expected value. The higher the tolerance factor, the wider this range. When the results of the aggregation stay within this range, the monitor doesn’t trigger. When the results of the aggregation cross this range, the monitor triggers. The tolerance factor can be any positive numeric value.
+    - The frequency is how often the monitor runs. This is a positive integer number of minutes.
+    - The range is the time range for your query. This is a positive integer number of minutes. The end time is the time the monitor runs.
```

Comment:
I don't understand the mention of end time here. Adding an additional line about the range. Two separate suggestions!

```suggestion
    - The range is the time range for your query. This is a positive integer number of minutes. A longer time range will allow the anomaly monitor to consider a larger number of datapoints when determining the expected behaviour of the query.
```